RepoCleanup/Functions/CreateOrgWithTeams.cs

```diff
@@ -0,0 +1,152 @@
+            }
+
+            logBuilder.Clear();
+
+            if (createdOrg)
+            {
+                Console.WriteLine($"Org {org.Username} successfully created.");
+            }
+            else
+            {
+                Console.WriteLine($"Create org for {org.Username} failed. See log {logName}.");
+            }
+
+            bool createdTeams = true;
+            List<CreateTeamOption> teams = new List<CreateTeamOption>();
+            teams.Add(CreateTeamInfo("Deploy-Prod", "Members can deploy to production", false, Permission.read));
+            teams.Add(CreateTeamInfo("Deploy-TT02", "Members can deploy to TT02", false, Permission.read));
+            teams.Add(CreateTeamInfo("Devs", "All application developers", true, Permission.write));
+
+            Console.WriteLine("Creating teams for org. Please wait.");
+            foreach (CreateTeamOption team in teams)
+            {
+                response = await GiteaService.CreateTeam(org.Username, team);
+
+                if (response.Success)
+                {
+                    logBuilder.AppendLine($"{DateTime.Now} - Information - Team {team.Name} was successfully added to org: {org.Username}.");
+                }
+                else
+                {
+                    createdTeams = false;
+                    logBuilder.AppendLine($"{DateTime.Now} - Error - Team {team.Name} was not added to org {org.Username}.");
+                    logBuilder.AppendLine($"{DateTime.Now} - Error -  {response.StatusCode}:{JsonSerializer.Serialize(response.ResponseMessage, Globals.SerializerOptions)}.");
+                }
+            }
+
+            if (createdTeams)
+            {
+                Console.WriteLine($"All teams for {org.Username} successfully created.");
+            }
+            else
+            {
+                Console.WriteLine($"Create teams for {org.Username} failed. See log {logName}.");
+            }
+        }
+
+        private static Organisation CollectOrgInfo()
+        {
+            bool isValid = false;
+            string username = string.Empty;
+
+            while (!isValid)
+            {
+                Console.Write("\r\nSet username (shortname) for org: ");
+                username = Console.ReadLine();
+
+                isValid = Regex.IsMatch(username, "^[a-z]+$");
+                if (!isValid)
+                {
+                    Console.WriteLine("Invalid name. Letters a-z are permitted.");
+                }
+            }
+
+            string fullname = string.Empty;
+
+            Console.Write("\r\nSet fullname for org: ");
+            fullname = Console.ReadLine();
+
+            isValid = false;
+            string website = string.Empty;
+
+            while (!isValid)
+            {
+                Console.Write("\r\nSet website for org: ");
+                website = Console.ReadLine();
+
+                isValid = Regex.IsMatch(website, "^[a-zA-Z0-9\\-._/:]*$");
```

Comment:
Would not adding a url be valid? I guess we might not always know it, and it doesn't really add any value having the website in gitea either.

Reply:
Changed. 
If you add a url it will be checked with regex.